docs/src/main/docs/about/02_introduction.adoc

```diff
@@ -0,0 +1,63 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2018, 2019 Oracle and/or its affiliates. All rights reserved.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= Introduction
+:description: about Helidon
+:keywords: helidon, java, microservices, microprofile
+
+== A collection of Java libraries
+
+Helidon is a collection of Java libraries for writing microservices.
+
+There is no specific tooling or deployment model, you simply use your favorite
+ tools !
+
```

Comment:
There's no need for any specific tooling or deployment model. Simply use your favorite tools !